rtt_rosparam/src/rtt_rosparam_service.cpp

```diff
@@ -239,6 +247,43 @@ class ROSParamService: public RTT::Service
   bool setParamComponentPrivate(const std::string &name) { return set(name, COMPONENT_PRIVATE); }
   bool setParamComponentRelative(const std::string &name) { return set(name, COMPONENT_RELATIVE); }
   bool setParamComponentAbsolute(const std::string &name) { return set(name, COMPONENT_ABSOLUTE); }
+
+  /**
+   * Adds a ROS parameter of any type as a property to this bag.
+   * A Property is created which causes contents of the
+   * property always to be in sync
+   * with the contents of the ROS parameter.
+   * @param name The name of this property
+   * @return the Property object by reference, which you can further query or document.
+   */
+  template<typename T, ResolutionPolicy P>
+  Property<T>& addRosParamProperty( const std::string& name) {
+  const std::string resolved_name = resolvedName(name,P);
+  typename internal::AssignableDataSource<T>::shared_ptr datasource( new RTT::rosparam::RosParamDataSource<T>(resolved_name) );
+    // if (this->getOwner() == nullptr) {
+    //   return nullptr;
+    // }
+    // if (this->getOwner()->properties() == nullptr) {
+    //   return nullptr;
+    // }
```

Comment:
```suggestion
```

remove dead code
`rosparam` can only be loaded into a component, so `getOwner()` and `getOwner()->properties()` must not return `nullptr`. If they would, also other calls in this class would cause segfaults.

And this repo must compile without C++11 enabled, so please do not use `nullptr` or other C++1x features.